sdk/python/feast/registry.py

```diff
@@ -104,6 +104,7 @@ def __init__(self, registry_config: RegistryConfig, repo_path: Path):
             repo_path: Path to the base of the Feast repository
             or where it will be created if it does not exist yet.
         """
+        self.lock = Lock()
```

Comment:
Should we have a more descriptive name for this variable, like `refresh_lock`. And can you please add `_` as a prefix?